### PR TITLE
updates to enable local packaging of alfalfa_worker

### PR DESCRIPTION
**Pull request by corymosiman12**
This resolves issue #51.  Even though the `TestSubmitFile` still does not run, it is no longer due to being unable to import alfalfa_worker modules, it has to do with issue  #50 still needs fixing.

### CODE REVIEW

**Comment by corymosiman12** [comment]
I believe now #50 should also be resolved with https://github.com/NREL/alfalfa/commit/1ffeb59cf38fad3fba4b8a3007d48202d2aac45d

File: alfalfa_worker/requirements/production.txt [hunk 1]
```diff
@@ -16,17 +15,18 @@ mlep==0.1.dev2
 more-itertools==5.0.0
 nodeenv==1.3.4
 pandas==0.24.2
+pycodestyle==2.5.0
```
Comment (nllong) [inline alfalfa_worker/requirements/production.txt:18]: these are defined in the test.txt requirements file. I'll fix this.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -15,6 +15,7 @@ repos:
     hooks:
       - id: trailing-whitespace
       - id: check-added-large-files
+        args: ['--maxkb=1700']
```
Comment (nllong) [inline .pre-commit-config.yaml:18]: nice!

**Comment by corymosiman12** [comment]
https://github.com/NREL/alfalfa/pull/52/commits/8c7afa4d11f7ca37f56ba13cdbc2c171bfcd6ee6 does a local install of of alfalfa_worker when building the docker container.

**Pull request merged**